app/server/static/components/question_and_answer.vue

```diff
@@ -0,0 +1,78 @@
+<template lang="pug">
+extends ./annotation.pug
+
+block annotation-area
+  div.card
+    header.card-header
+      div.card-header-title.has-background-royalblue
+        div.field.is-grouped.is-grouped-multiline
+          div.control
+            div.tags.has-addons.white(
+              v-if="docs[pageNumber] && docs[pageNumber].extra_text"
+              ) {{ docs[pageNumber].extra_text || 'missing quesiton' }}
+
+    div.card-content
+      div.content(v-if="docs[pageNumber] && annotations[pageNumber]")
+        annotator(
+          v-bind:labels="labels"
+          v-bind:entity-positions="annotations[pageNumber]"
+          v-bind:text="docs[pageNumber].text"
+          v-on:remove-answer="removeLabel"
+          v-on:add-answer="addAnswer"
+          ref="annotator"
+        )
+    div.card-content.has-text-centered
+      button.button.is-link.is-medium(
+        v-bind:disabled="this.hasAnswer"
+        v-on:click="annotate()") Submit Selection
```

Comment:
Nit: to be consistent with other formatting, this should look like so:

```pug
      button.button.is-link.is-medium(
        v-bind:disabled="this.hasAnswer"
        v-on:click="annotate()"
      ) Submit Selection

```

Reply:
true